source/guide_goaccess.rst

```diff
@@ -0,0 +1,199 @@
+.. highlight:: console
+
+.. author:: FM <git.fm@mmw9.de>
+
+########
+GoAccess
+########
+
+GoAccess_ is an open source realtime web log analyzer and viewer. It use the standard web server log file as source to build a readable graphical HTML output as web page (dashboard) or an overview in the shell. 
+
+This documentation describe the way to let create a static web page as overview, which will be updated every hour with a cron job. More details are on the `man page`_ and on GitHub_ available.
+
+License
+=======
+
+GoAccess is released under the `MIT License`_
+
+Installation
+============
+
+Step 1 - Download and Extract the Source Code
+---------------------------------------------
+
+Create a working directory.
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/filebase/
+ [isabell@stardust ~]$ cd ~/filebase/
+ [isabell@stardust ~]$
+
+Download the actual source code from GitHub. 
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/allinurl/goaccess.git
```

Comment:
What do you think about using [the tarball](https://goaccess.io/download#installation) instead? This way we can skip `autoreconf`.